packages/devtools/lib/src/utils.dart

```diff
@@ -7,10 +7,14 @@ import 'dart:convert';
 import 'dart:io';
 import 'dart:math';
 
+import 'package:collection/collection.dart';
 import 'package:intl/intl.dart';
 import 'package:path/path.dart' as path;
 import 'package:vm_service_lib/vm_service_lib.dart';
 
+bool collectionEquals<T>(T e1, T e2) =>
```

Comment:
can this be:
<T extends Collection>
?

Reply:
reverted to dynamic since we won't benefit from type checking here